python/mdvtools/project_router.py

```
@@ -1,11 +1,16 @@
-import os
-from flask import Flask, Response, request, jsonify, session, redirect, url_for
+import logging
+from flask import Flask, request, jsonify, session
```

Ruff (0.8.2)

`flask.request` imported but unused

Remove unused import: `flask.request`

(F401)